readthedocs/projects/views/private.py

```diff
@@ -458,14 +453,6 @@ def get_form(self, data=None, files=None, **kwargs):
         kwargs['user'] = self.request.user
         return super().get_form(data, files, **kwargs)
 
-    def form_valid(self, form):
-        broadcast(
-            type='app',
-            task=tasks.symlink_subproject,
```

Comment:
I think a lot of the places we were calling symlink code we actually want to bust the cache for the project as well. Something to consider, though maybe we want to _add_ that logic in a new PR after this.

Reply:
I agree. In _this_ particular case it was called when de-activating a version. The following line (which I didn't add) triggers a `clean_project_resources` which removes the version from the storage. Seems like someone already thought about this :smile:

Reply:
It seems I already added some ones where I found the same case: https://github.com/readthedocs/readthedocs.org/pull/6535/files#diff-6e7074d6dee951393c702a7071bc0f1eR232